examples/text_labels_and_annotations/mathtext_fontset_example.py

```diff
@@ -0,0 +1,77 @@
+"""
+========================
+Mathtext Fontset Example
+========================
+
+"""
+
+import matplotlib.pyplot as plt
+
+plt.figure(figsize=(20, 10))
```

Comment:
Please try to make the figure not wider than 9. Otherwise, the rendered figure in the docs will be downscaled to fit in the available space on the HTML page, which usually does not look good.